BDT_application/step2_BDTapplication_condor/TMVAClassificationApplication.C

```diff
@@ -1168,19 +1199,19 @@ TH1F* Getoutput( TString myMethodList = "", std::string input_name="",float xs=1
 	   ctag_norm+=genweight*norm_scale*lumi*mu_id*ele_id*trig_SF*charFlip_SF*sig_pdfdo;
 	 }
 	 else if(weight_name=="sig_scaleup"){
-	   histBdtG->Fill( reader->EvaluateMVA( "BDTG method"), genweight*norm_scale*lumi*mu_id*ele_id*trig_SF*charFlip_SF*sig_scaleup*ctag_SF_LHEScaleWeightmuFup*ctag_SF_LHEScaleWeightmuRup);
+	   histBdtG->Fill( reader->EvaluateMVA( "BDTG method"), genweight*norm_scale*lumi*mu_id*ele_id*trig_SF*charFlip_SF*sig_scaleup*ctag_SF_LHEScaleWeightmuFup*ctag_SF);
 	   ctag_norm+=genweight*norm_scale*lumi*mu_id*ele_id*trig_SF*charFlip_SF*sig_scaleup;
 	 }
 	 else if(weight_name=="sig_scaledo"){
-	   histBdtG->Fill( reader->EvaluateMVA( "BDTG method"), genweight*norm_scale*lumi*mu_id*ele_id*trig_SF*charFlip_SF*sig_scaledo*ctag_SF_LHEScaleWeightmuFdo*ctag_SF_LHEScaleWeightmuRdo);
+	   histBdtG->Fill( reader->EvaluateMVA( "BDTG method"), genweight*norm_scale*lumi*mu_id*ele_id*trig_SF*charFlip_SF*sig_scaledo*ctag_SF_LHEScaleWeightmuFdo*ctag_SF);
 	   ctag_norm+=genweight*norm_scale*lumi*mu_id*ele_id*trig_SF*charFlip_SF*sig_scaledo;
 	 }
 	 else if(weight_name=="sig_psup"){
-	   histBdtG->Fill( reader->EvaluateMVA( "BDTG method"), genweight*norm_scale*lumi*mu_id*ele_id*trig_SF*charFlip_SF*sig_psup*ctag_SF_PSWeightFSRup*ctag_SF_PSWeightISRup);
+	   histBdtG->Fill( reader->EvaluateMVA( "BDTG method"), genweight*norm_scale*lumi*mu_id*ele_id*trig_SF*charFlip_SF*sig_psup*ctag_SF_PSWeightFSRup*ctag_SF);
```

Comment:
May be I am confused, why the ``ctag_SF_PSWeightISRup`` drops?

Reply:
Because conveners realized and we also agreed that we should not correlate ctag theory uncertainties with our theory uncertainties. See the first pre-approval comment.

Reply:
But I see FSR is still there. So, now I am confused.

Reply:
I missed the FSR part, I should drop it.

Reply:
I see done now.

Reply:
It doesn't seem to be dropped?